features/steps/steps/propertysets_qtys_units.py

```diff
@@ -112,7 +112,7 @@ def normalize(name, case_sensitive=True):
         return name if case_sensitive else name.lower()
 
     schema_specific_path = system.get_abs_path(f"resources/{schema.upper()}/{table}.csv")
-    if os.path.exists(schema_specific_path):
+    if schema_specific_path is not None and os.path.exists(schema_specific_path):
```

Comment:
get_abs_path() uses glob, then index 0th element and catches index error then defaults to None.

So path.exists call is probably unnecessary, but the is not None check is not.

Some type hints and better descriptive function names would really help here, but not for now..